cpp/log/cert.cc

```diff
@@ -887,10 +963,23 @@ bool ValidateRedactionSubjectAltNames(STACK_OF(GENERAL_NAME) *
 }  // namespace
 
 
+util::Status Cert::SubjectAltNames(vector<string>* dns_alt_names) const {
+  ScopedGENERAL_NAMEStack subject_alt_names(
+      static_cast<STACK_OF(GENERAL_NAME)*>(X509_get_ext_d2i(
+          x509_.get(), NID_subject_alt_name, nullptr, nullptr)));
+
+  if (subject_alt_names) {
```

Comment:
Are you going to do it in this PR, or in a separate one? Just so I know to hold off my review until you're done with that change, I don't really mind either way.